### DISCUSSION TITLE
Is Tapo Light Bulb L535E supported?

### DISCUSSION THREAD

**Discussion by AndyRaphael** [petretiandrea/home-assistant-tapo-p100#680] [Q&A]
I see that the L530 is supported, but I was wondering if the L535E model is supported as well. It's Matter certified with Apple HomeKit support compared to the L530.

Thank you for writing this awesome integration and getting the Tapo line added to Home Assistant!

**Comment by petretiandrea** [comment]
Yes, I will add
